docs/events/mouse_click.md

```diff
@@ -0,0 +1,20 @@
+# Click
+
+The `Click` event is sent to a widget when the user clicks a mouse button.
+
+- [x] Bubbles
```

Comment:
All other `mouse_xxx.md` files seem to have a `Mousexxx` event. Is this event purposefully different, or should this be a `MouseClick` event?

Reply:
I see your point, but the mouse prefix is redundant for click. There is no other kind of click. I think I might have to be pragmatic and accept a little inconsistency.

Reply:
I suspected this was intentional but decided to raise the issue either way, just to be extra sure.